synapseclient/models/folder.py

```diff
@@ -330,14 +330,10 @@ async def store_async(
                 description=self.description,
             )
             delete_none_keys(synapse_folder)
-            entity = await loop.run_in_executor(
-                None,
-                lambda: Synapse.get_client(synapse_client=synapse_client).store(
-                    obj=synapse_folder,
-                    set_annotations=False,
-                    isRestricted=self.is_restricted,
-                    createOrUpdate=False,
-                ),
```

Comment:
This run_in_executor code is stuff I am trying to get rid it since it was the original hack used to wrap the original Synapse client.